services/tokei/tokei-github-lines.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { nonNegativeInteger } = require('../validators')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  lines: nonNegativeInteger,
+}).required()
+
+const documentation = `
+<p>
+  The <code>provider</code> is the domain name of git host.
+  If no TLD is provided, <code>.com</code> will be added.
+  For example, setting <code>gitlab</code> or <code>bitbucket.org</code> as the provider also works.
+</p>
+`
+
+module.exports = class LinesOfCode extends BaseJsonService {
+  static category = 'size'
+
+  static route = { base: 'tokei/l', pattern: ':provider/:user/:repo' }
+
+  static get examples() {
+    return [
+      {
+        title: 'Lines of code',
+        namedParams: {
+          provider: 'github',
+          user: 'badges',
+          repo: 'shields',
+        },
+        staticPreview: this.render({ lines: 119500 }),
+        keywords: ['loc', 'tokei'],
+        documentation,
+      },
+    ]
+  }
+
+  static defaultBadgeData = { label: 'total lines' }
```

Comment:
Since both the label and color will be static, we can just utilize default badge data. We should also use `blue` as the default color since this is informational, whereas brightgreen colored badges are used to indicate some kind of pass/success gate (build passing, high test/code coverage %, etc.) 
```suggestion
  static defaultBadgeData = {
    label: 'total lines',
    color: 'blue'
  }
```